wiki/deploy.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wiki
+  namespace: default
+  labels:
+    app: wiki
+  annotations:
+    fluxcd.io/automated: 'true'
+spec:
+  replicas: 1
+  revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      app: wiki
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      name: wiki
+      labels:
+        app: wiki
+    spec:
+      containers:
+      - name: wiki
+        image: requarks/wiki:2
+        imagePullPolicy: IfNotPresent
```

Comment:
Dit is een slecht idee als je een image tag gebruikt die ge-update wordt, maar kan blijven als je Flux gebruikt voor updates.

Reply:
Het is denk ik opzich best handig dat je auto-updates krijgt

Reply:
Dat is het zeker, maar met `IfNotPresent` krijg je nooit updates als het image al op de host beschikbaar is. Hierdoor kun je op twee hosts verschillende images krijgen, en dat gaat sowieso voor verrassingen zorgen.